### DISCUSSION TITLE
how set the proxy for alert wecha webhook

### DISCUSSION THREAD

**Discussion by unihon** [apache/skywalking#11762] [Q&A]
linux env;
``` 
export http_proxy=xx
export https_proxy=xx
```

oapService.sh jvm config
```
java -Dhttp.proxyHost=xx -Dhttp.proxyPort=xx  ...
```

Neither works as expected

**Comment by wu-sheng** [comment]
Sorry, the upstream never tested proxy mode. It should relate to your OS, JVM and HTTP lib(Armeria).
We don't know that.
